mesa/agent.py

```diff
@@ -90,7 +90,7 @@ class AgentSet(MutableSet, Sequence):
 
     Methods:
         __len__, __iter__, __contains__, select, shuffle, sort, _update, do, get, __getitem__,
-        add, discard, remove, __getstate__, __setstate__, random
+        add, discard, remove, __getstate__, __setstate__, random, group_by
```

Comment:
I guess this is from another branch?

Reply:
I think thats was a mistake by me. Let me fix it :)